local/iatlas/cbioportal_export/clinical_to_cbioportal.py

```diff
@@ -576,50 +621,72 @@ def save_to_synapse(
 
 
 def validate_export_files(
```

Comment:
Does cbioportal validator do these kind of checks?

Reply:
Unsure regarding the value of "NA" in patient or sample ids but they shouldn't do checks for any of the other ones. - A study can be considered PASS in the cbioportal validator with just having clinical files for example so we want to make sure we have all files present. 
- Also want to check that we didn't lose any PATIENT_IDs and/or SAMPLE_IDs in this pipeline.